src/psyclone/dynamo0p3.py

```diff
@@ -6059,13 +7135,19 @@ def operator(self, arg):
         self._arglist.append(arg.proxy_name_indexed+"%local_stencil")
 
     def cma_operator(self, arg):
-        ''' add the CMA operator and associated scalars to the argument
-        list '''
+        '''
+        Add the CMA operator and associated scalars to the argument
+        list.
+
+        :param arg: the CMA operator argument.
+        :type arg: :py:class:`psyclone.dynamo0p3.DynKernelArgument`
+
+        '''
         if arg.function_space_to.orig_name != \
            arg.function_space_from.orig_name:
-            components = ["matrix"] + DynInvokeCMAOperators.cma_diff_fs_params
+            components = ["matrix"] + DynCMAOperators.cma_diff_fs_params
```

Comment:
You could set components = ["matrix"] before the if/else and then do += inside the if/else.

Reply:
Done.